fpo-web/src/app/app-routing.module.ts

```diff
@@ -57,13 +45,13 @@ const routes: Routes = [
     path: 'glossary-editor',
     redirectTo: 'prv/glossary-editor'
   },
-  {
-    path: 'prv/glossary-editor',
-    component: GlossaryEditorComponent,
-    data: {
-      breadcrumb: 'Glossary Editor',
-    }
-  },
+  // {  //commenting out unnecessary router for our current main goal.
```

Comment:
This comment doesn't fully describe why  or what  is being commented out. 

'Main goal' will have zero context/relevance 1 month down the road or 1 year down the road. 

'This route has been disabled. It is a feature that was apart of the originating fork of the repo [repo link] and is not being leveraged. The route exposes a feature [blah]. '